web/handlers.go

```diff
@@ -97,26 +87,49 @@ func (h Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	token, tokenLocation := app.ParseAuthTokenFromRequest(r)
+
 	if len(token) != 0 {
 		session, err := c.App.GetSession(token)
+		csrfPassed := false
+
+		// CSRF Check
+		// ToDo(DSchalla) 2019/01/04: Confirm no state is modified with GET requests in the product
```

Comment:
`Confirm no state is modified with GET requests in the product` Yes, these changes require that this be the case. Don't think this is the right place to put that TODO though.

Reply:
Removed ToDo and created new ticket